docs/reference/announcements-release-notes/880/880-release-notes.md

```diff
@@ -609,17 +622,21 @@ You can now save and rerun versioned test scenarios in Web Modeler:
 - Ideal for process developers and CoEs
 - Enables behavior-driven development with Camunda Process Test
 
+<p><a href="../../../../components/modeler/web-modeler/play-your-process/#scenarios" class="link-arrow">Scenarios</a></p>
+
 ### RDBMS support for Oracle and MS SQL in Self-Managed
 
 <!-- https://github.com/camunda/product-hub/issues/2558 -->
 
 Web Modeler Self-Managed now supports Oracle Database and Microsoft SQL Server for simpler setup and maintenance.
 
+<p><a href="../../../../self-managed/components/modeler/web-modeler/configuration/database/#using-alternative-database-vendors" class="link-arrow">Set up Oracle or MS SQL</a></p>
+
 ### Username claim configuration
 
 <!-- https://github.com/camunda/web-modeler/issues/9292 -->
 
-Configure the claim used for usernames via the `CAMUNDA_IDENTITY_USERNAMECLAIM` environment variable for Tasklist, Optimize, and Operate authentication.
```

Comment:
"Tasklist, Optimize, and Operate authentication" did not really make sense in the Web Modeler context.